docs/.vuepress/config.js

```diff
@@ -232,7 +232,6 @@ module.exports = {
               '/commands/regen_tendermint',
               '/commands/regen_testnet',
               '/commands/regen_tx',
-              '/commands/regen_unsafe-reset-all',
```

Comment:
CLI commands are aligned with master and this command is now a subcommand to `tendermint`.